core/src/main/resources/__flow__/types/global.fjs

```diff
@@ -3,6 +3,8 @@
 declare type Services = { ophan: OphanService
                         , dom: DomService
                         , viewport: ViewportService
+                        , consent: { acast: boolean; }
```

Comment:
Note, this is not a callback, so we may miss consent here depending on when the Frontend JS for consent executes relative to the atom rendering. I've done this for simplicity but we may want to improve this depending on numbers/revenue implications.

Reply:
Update: the CMP boostraps *after* atoms on Frontend so this will occur for first-time visits. However, given `atom-renderer` is on borrowed time I'm comfortable with a temporary suboptimal solution here.